### ISSUE TITLE
Rule configures an output path to a drive that doesn't exist 

### ISSUE THREAD

**Issue by Frizi** [bazel-contrib/setup-bazel#46]
I'm attempting to run a bazel build under a self-hosted windows runner. That runner doesn't have a `D:\` drive present. Despite that, the rule hardcodes the bazel output path and all cache locations to `D:\` drive, resulting in following error at runtime:
```
FATAL: Output base directory 'd:\_bazel' could not be created: (error: 3): The system cannot find the path specified.
```

It would be very handy if at least the windows drive letter was configurable.

**Comment by p0deje** [comment]
@Frizi can you test if the following config works for you?

```yaml
- uses: bazel-contrib/setup-bazel@0.9.0
  with:
    bazelrc: startup --output_base=c:/_bazel
```

**Comment by Frizi** [comment]
I doubt that will have an appropriate behaviour once you enable caching. My config currently looks like that:
```yaml
      - uses: bazel-contrib/setup-bazel@0.9.0
        with:
          bazelisk-cache: true
          disk-cache: true
          repository-cache: true
```

Since those [hardcoded paths](https://github.com/bazel-contrib/setup-bazel/blob/f3f50ea6791b9b0f4c4eeabba4507422426462f5/config.js#L26-L28) are passed as is to all the cache handlers, I believe the that manual config line will only cause them to be misaligned with paths actually used by bazel.

**Comment by p0deje** [comment]
@Frizi Do you think the action should fall back to drive `C:` if `D:` is not available to have a seamless experience or provide a configuration option?

**Comment by Frizi** [comment]
I'm happy with either one, but a drive letter configuration option seems to me like a more robust option, and likely easier to implement as well.

**Comment by p0deje** [comment]
Can you try using https://github.com/bazel-contrib/setup-bazel/commit/09f3a72d13a081857b0ee94e986ffa84caef7c85 and see if it works for you? I had no chance to test it yet.

**Comment by Frizi** [comment]
That appears to work for us now. Thanks a lot!

**Comment by p0deje** [comment]
Great, I've released 0.10.0 with the change
